contracts/ERC677BridgeToken.sol

```diff
@@ -68,9 +66,8 @@ contract ERC677BridgeToken is IBurnableMintableERC677Token, DetailedERC20, Burna
     }
 
     function callAfterTransfer(address _from, address _to, uint256 _value) internal {
```

Comment:
Consider to add comment describing that we don't follow the erc677 standard only for the bridge contract.

Reply:
Sure, did it.